bookwyrm/templates/book/format_info.html

```diff
@@ -0,0 +1,39 @@
+{% spaceless %}
+
+{% load i18n %}
+{% load book_display_tags %}
+
+{% firstof book.physical_format_detail book.get_physical_format_display as format %}
+{% firstof book.physical_format book.physical_format_detail as format_property %}
+{% with pages=book.pages %}
+{% if format or pages %}
+
+{% if format_property %}
+<meta itemprop="bookFormat" content="{{ format_property }}">
+{% if book.physical_format == "AudiobookFormat" and book.audiobook_play_time %}
+    <meta itemprop="timeRequired" content="{{ book.audiobook_play_time|iso_duration }}">
+{% endif %}
+{% endif %}
+
+{% if pages %}
+<meta itemprop="numberOfPages" content="{{ pages }}">
+{% endif %}
+
+<p>
+    {% if format and not pages %}
```

Comment:
This does mean that if a book is marked as an audiobook but has both playtime and pages, it will show pages. That's kind of an edge case, and it might make the boolean logic super complicated to address, though

Reply:
I agree that it would complicate things, and is rarely bound to happen.